models/staging/raw_customers.py

```diff
@@ -1,6 +1,19 @@
 import pandas as pd
+from sshfs import SSHFileSystem
+
 
 def model(dbt, session) -> pd.DataFrame:
     dbt.config(materialized="table")
-    df = pd.read_csv("http://35.196.10.176:8000/customers/data.csv")
+    dbt.config(fal_environment="sftp")
```

Comment:
How does two `dbt.config` calls behave? Should it be 

```suggestion
    dbt.config(
      materialized="table",
      fal_environment="sftp"
    )
```